repos/system_upgrade/el7toel8/actors/scanmemory/tests/test_scanmemory.py

```diff
@@ -0,0 +1,24 @@
+import mock
+
+from leapp.libraries.actor import library
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+def test_with_low_memory(monkeypatch):
+    with mock.patch("__builtin__.open", mock.mock_open(read_data="MemTotal: 42 kB")) as mock_proc_meminfo:
```

Comment:
Open question: why are you checking for MemTotal?

Reply:
if I understood correctly your question, the goal for these actor is to check the total memory of the machine.. that's the reason I'm checking that.

Reply:
Why are you relying on MemTotal, not MemFree or MemAvailable?

Reply:
@fernflower Because it is HW requirement check. So you are not interested about info how much free or available memory you have.

Reply:
@pirat89 ah, ok, that makes sense then. Thanks!

Reply:
Reading the document (https://access.redhat.com/articles/rhel-limits), I understood minimum required memory as total memory on the machine to rhel8 run, not Available/Free, because it's depends what's is running at the time. 

But is possible I miss understood that...

Reply:
@fellipeh no that's fine, that was only a question. Figured out if I have it others may wonder as well - so won't hurt to clear that once and for all.